src/utils.js

```diff
@@ -19,24 +20,45 @@ const exec = async (command, opts) => {
   });
 };
 
+const mime_type = async opts => {
+  const { path, buffer } = opts;
+
+  try {
+    const svg_cadidate = path
```

Comment:
typo: cadidate -> candidate

Reply:
I wonder if we should just rely on a file extension instead of reading it? (if it's a path) ... just to avoid reading into memory. And potentially it should be enough, I guess.

Reply:
> wonder if we should just rely on a file extension instead of reading it?

I have added that check to not have to read it in memory